packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3480,6 +3484,14 @@ class InputDecoration {
   /// a default height based on text size.
   final BoxConstraints? constraints;
 
+  /// Optional widget that appears below the [InputDecorator.child] and the border.
+  /// If non-null, the border's color animates to red and the [helperText] is
+  /// not shown.
```

Comment:
nit: blank line after this first sentence, so that it becomes a paragraph of its own:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements